tests/build_image_base_test.py

```diff
@@ -66,6 +66,17 @@ def test_common_packages(self):
         self.assertTrue(self.is_package_present("aws"))
         self.assertTrue(self.is_package_present("jq"))
 
+    def test_non_root_user(self):
+        """
+        Test using non-root `sam` user
+        """
+        output = (
+            self.client.containers.run(image=self.image, user="sam", command="whoami")
+            .decode()
+            .strip()
+        )
+        self.assertEqual(output, "sam")
```

Comment:
Do we need a test that tries to run a command but fails without root permissions?

Reply:
Good catch, I changed so it checks that UID is not 0 (which is [always 0 for the root user](https://superuser.com/questions/626843/does-the-root-account-always-have-uid-gid-0)).